index.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 from flask import  Flask
+from werkzeug.wrappers import Response as ResponseBase
```

Comment:
optional: Flask itself already provides a wrapper around this `werkzeug.wrappers. Response `, which can be imported by `from flask import Response`. But this also works!